src/__tests__/services/mod-translation-check.test.ts

```
@@ -0,0 +1,252 @@
+      id: 'custommod',
+      name: 'Custom Mod',
+      path: '/mods/custommod-1.0.0.jar',
+      expectedTranslations: {
+        ja_jp: false,
+        zh_cn: false,
+        ko_kr: false,
+        de_de: false,
+      },
+    },
+  ];
+};
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)